packages/react-scripts/config/webpack.config.js

```diff
@@ -278,7 +278,11 @@ module.exports = function(webpackEnv) {
       //   name: false,
       // },
       splitChunks: bpkReactScriptsConfig.enableAutomaticChunking
-        ? { chunks: 'all', name: true }
+        ? {
+            chunks: 'all',
+            name: true,
```

Comment:
Now, I understand that this is unrelated to your PR, but I reckon we should change `name: true` to `name: false` like it previously was (see line 278).

I changed this back in the day not knowing this caveat:
> It is recommended to set splitChunks.name to false for production builds so that it doesn't change names unnecessarily.
(https://webpack.js.org/plugins/split-chunks-plugin#splitchunksname)

Essentially it takes `vendor` / `main` out of the file name and instead relies on a hash to improve long term caching.

Reply:
I can do that